tests/templates/__snapshots__/RegularGivingConfirmationTest__testItRenders__1.txt

```diff
@@ -0,0 +1,65 @@
+    <h1>Regular Giving Mandate for [Charity Name]</h1>
+
+    <p>
+    Thank you for supporting [Charity Name] with your generous gift through Big Give.
+    </p>
+
+    <p>
+    A message from [Charity Name]:
+    </p>
+    <blockquote>[Campaign Thank You Message]</blockquote>
+
+    <p>
+        campaign@thankyou.email
+    </p>
+
+    <h2>Regular Giving Mandate Details</h2>
+    <p>
+        Thank you for setting up your regular gift to [Charity Name] for the [Campaign Name] campaign. [Charity Name]
+        is registered with the Charity Commission for England and Wales with charity number [Charity Number]. This email
+        certifies that no goods or services will be provided in return for your contribution.
+    </p>
+
+    <ul>
+        <li>Regular gift sign up date: DD/MM/YYYY HH:MM</li>
+        <li>Donor: [Donor Name]</li>
+        <li>Schedule: Monthly on day #3</li>
+        <li>Next expected payment date: MM/DD/YYYY</li>
+        <li>Amount: £20.00</li>
+        <li>Gift aid value: £4.00</li>
+        <li>Total with gift aid: £24.00</li>
+    </ul>
+    <p>Total amount you have been charged is £20.00</p>
```

Comment:
this line copied from suggestion on Jira but I'm not sure it make sense here - maybe better moved into the first donation details section?